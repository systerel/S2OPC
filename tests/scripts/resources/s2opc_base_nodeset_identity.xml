<?xml version='1.0' encoding='utf-8'?>
<UANodeSet xmlns="http://opcfoundation.org/UA/2011/03/UANodeSet.xsd" xmlns:uax="http://opcfoundation.org/UA/2008/02/Types.xsd" LastModified="2022-03-29T00:00:00Z">
  <Models>
    <Model ModelUri="http://opcfoundation.org/UA/" XmlSchemaUri="http://opcfoundation.org/UA/2008/02/Types.xsd" Version="1.04.11" PublicationDate="2022-03-29T00:00:00Z" />
  </Models>
  <Aliases>
    <Alias Alias="Boolean">i=1</Alias>
    <Alias Alias="SByte">i=2</Alias>
    <Alias Alias="Byte">i=3</Alias>
    <Alias Alias="Int16">i=4</Alias>
    <Alias Alias="UInt16">i=5</Alias>
    <Alias Alias="Int32">i=6</Alias>
    <Alias Alias="UInt32">i=7</Alias>
    <Alias Alias="Int64">i=8</Alias>
    <Alias Alias="UInt64">i=9</Alias>
    <Alias Alias="Float">i=10</Alias>
    <Alias Alias="Double">i=11</Alias>
    <Alias Alias="DateTime">i=13</Alias>
    <Alias Alias="String">i=12</Alias>
    <Alias Alias="ByteString">i=15</Alias>
    <Alias Alias="Guid">i=14</Alias>
    <Alias Alias="XmlElement">i=16</Alias>
    <Alias Alias="NodeId">i=17</Alias>
    <Alias Alias="ExpandedNodeId">i=18</Alias>
    <Alias Alias="QualifiedName">i=20</Alias>
    <Alias Alias="LocalizedText">i=21</Alias>
    <Alias Alias="StatusCode">i=19</Alias>
    <Alias Alias="Structure">i=22</Alias>
    <Alias Alias="Number">i=26</Alias>
    <Alias Alias="Integer">i=27</Alias>
    <Alias Alias="UInteger">i=28</Alias>
    <Alias Alias="HasComponent">i=47</Alias>
    <Alias Alias="HasProperty">i=46</Alias>
    <Alias Alias="Organizes">i=35</Alias>
    <Alias Alias="HasEventSource">i=36</Alias>
    <Alias Alias="HasNotifier">i=48</Alias>
    <Alias Alias="HasSubtype">i=45</Alias>
    <Alias Alias="HasTypeDefinition">i=40</Alias>
    <Alias Alias="HasModellingRule">i=37</Alias>
    <Alias Alias="HasEncoding">i=38</Alias>
    <Alias Alias="HasDescription">i=39</Alias>
    <Alias Alias="HasCause">i=53</Alias>
    <Alias Alias="ToState">i=52</Alias>
    <Alias Alias="FromState">i=51</Alias>
    <Alias Alias="HasEffect">i=54</Alias>
    <Alias Alias="HasTrueSubState">i=9004</Alias>
    <Alias Alias="HasFalseSubState">i=9005</Alias>
    <Alias Alias="HasDictionaryEntry">i=17597</Alias>
    <Alias Alias="HasCondition">i=9006</Alias>
    <Alias Alias="HasGuard">i=15112</Alias>
    <Alias Alias="HasAddIn">i=17604</Alias>
    <Alias Alias="HasInterface">i=17603</Alias>
  </Aliases>
  <UADataType NodeId="i=24" BrowseName="BaseDataType" IsAbstract="true">
    <DisplayName>BaseDataType</DisplayName>
    <References>
      <Reference ReferenceType="Organizes" IsForward="false">i=90</Reference>
    </References>
  </UADataType>
  <UAObjectType NodeId="i=58" BrowseName="BaseObjectType">
    <DisplayName>BaseObjectType</DisplayName>
    <References>
      <Reference ReferenceType="HasSubtype">i=61</Reference>
      <Reference ReferenceType="HasSubtype">i=75</Reference>
    </References>
  </UAObjectType>
  <UAObjectType NodeId="i=61" BrowseName="FolderType">
    <DisplayName>FolderType</DisplayName>
    <References>
      <Reference ReferenceType="HasSubtype" IsForward="false">i=58</Reference>
      <Reference ReferenceType="HasTypeDefinition" IsForward="false">i=84</Reference>
      <Reference ReferenceType="HasTypeDefinition" IsForward="false">i=85</Reference>
      <Reference ReferenceType="HasTypeDefinition" IsForward="false">i=86</Reference>
      <Reference ReferenceType="HasTypeDefinition" IsForward="false">i=90</Reference>
    </References>
  </UAObjectType>
  <UAVariableType NodeId="i=62" BrowseName="BaseVariableType" IsAbstract="true" ValueRank="-2">
    <DisplayName>BaseVariableType</DisplayName>
    <References>
      <Reference ReferenceType="HasSubtype">i=63</Reference>
    </References>
  </UAVariableType>
  <UAVariableType NodeId="i=63" BrowseName="BaseDataVariableType" ValueRank="-2">
    <DisplayName>BaseDataVariableType</DisplayName>
    <References>
      <Reference ReferenceType="HasSubtype" IsForward="false">i=62</Reference>
      <Reference ReferenceType="HasSubtype">i=72</Reference>
    </References>
  </UAVariableType>
  <UAVariableType NodeId="i=72" BrowseName="DataTypeDictionaryType" ReleaseStatus="Deprecated" DataType="ByteString">
    <DisplayName>DataTypeDictionaryType</DisplayName>
    <References>
      <Reference ReferenceType="HasSubtype" IsForward="false">i=63</Reference>
      <Reference ReferenceType="HasTypeDefinition" IsForward="false">i=7617</Reference>
    </References>
  </UAVariableType>
  <UAObjectType NodeId="i=75" BrowseName="DataTypeSystemType" ReleaseStatus="Deprecated">
    <DisplayName>DataTypeSystemType</DisplayName>
    <References>
      <Reference ReferenceType="HasSubtype" IsForward="false">i=58</Reference>
      <Reference ReferenceType="HasTypeDefinition" IsForward="false">i=93</Reference>
    </References>
  </UAObjectType>
  <UAObject NodeId="i=84" BrowseName="Root" SymbolicName="RootFolder">
    <DisplayName>Root</DisplayName>
    <References>
      <Reference ReferenceType="HasTypeDefinition">i=61</Reference>
      <Reference ReferenceType="Organizes">i=85</Reference>
      <Reference ReferenceType="Organizes">i=86</Reference>
    </References>
  </UAObject>
  <UAObject NodeId="i=85" BrowseName="Objects" SymbolicName="ObjectsFolder">
    <DisplayName>Objects</DisplayName>
    <References>
      <Reference ReferenceType="Organizes" IsForward="false">i=84</Reference>
      <Reference ReferenceType="HasTypeDefinition">i=61</Reference>
      <Reference ReferenceType="Organizes">i=2253</Reference>
    </References>
  </UAObject>
  <UAObject NodeId="i=86" BrowseName="Types" SymbolicName="TypesFolder">
    <DisplayName>Types</DisplayName>
    <References>
      <Reference ReferenceType="Organizes" IsForward="false">i=84</Reference>
      <Reference ReferenceType="HasTypeDefinition">i=61</Reference>
      <Reference ReferenceType="Organizes">i=90</Reference>
    </References>
  </UAObject>
  <UAObject NodeId="i=90" BrowseName="DataTypes" SymbolicName="DataTypesFolder">
    <DisplayName>DataTypes</DisplayName>
    <References>
      <Reference ReferenceType="Organizes" IsForward="false">i=86</Reference>
      <Reference ReferenceType="Organizes">i=24</Reference>
      <Reference ReferenceType="HasTypeDefinition">i=61</Reference>
      <Reference ReferenceType="Organizes">i=93</Reference>
    </References>
  </UAObject>
  <UAObject NodeId="i=93" BrowseName="OPC Binary" SymbolicName="OPCBinarySchema_TypeSystem" ReleaseStatus="Deprecated">
    <DisplayName>OPC Binary</DisplayName>
    <References>
      <Reference ReferenceType="Organizes" IsForward="false">i=90</Reference>
      <Reference ReferenceType="HasTypeDefinition">i=75</Reference>
      <Reference ReferenceType="HasComponent">i=7617</Reference>
    </References>
  </UAObject>
  <UAVariable NodeId="i=7617" BrowseName="Opc.Ua" SymbolicName="OpcUa_BinarySchema" ReleaseStatus="Deprecated" DataType="ByteString">
    <DisplayName>Opc.Ua</DisplayName>
    <References>
      <Reference ReferenceType="HasComponent" IsForward="false">i=93</Reference>
      <Reference ReferenceType="HasTypeDefinition">i=72</Reference>
    </References>
  </UAVariable>
  <UAObject NodeId="i=2253" BrowseName="Server" EventNotifier="1">
    <DisplayName>Server</DisplayName>
    <References>
      <Reference ReferenceType="HasProperty">i=2254</Reference>
      <Reference ReferenceType="HasProperty">i=2255</Reference>
      <Reference ReferenceType="Organizes" IsForward="false">i=85</Reference>
    </References>
  </UAObject>
  <UAVariable NodeId="i=2254" BrowseName="ServerArray" ParentNodeId="i=2253" DataType="String" ValueRank="1" ArrayDimensions="0" MinimumSamplingInterval="1000">
    <DisplayName>ServerArray</DisplayName>
    <References>
      <Reference ReferenceType="HasProperty" IsForward="false">i=2253</Reference>
    </References>
  </UAVariable>
  <UAVariable NodeId="i=2255" BrowseName="NamespaceArray" ParentNodeId="i=2253" DataType="String" ValueRank="1" ArrayDimensions="0" MinimumSamplingInterval="1000">
    <DisplayName>NamespaceArray</DisplayName>
    <References>
      <Reference ReferenceType="HasProperty" IsForward="false">i=2253</Reference>
    </References>
    <Value>
      <uax:ListOfString>
        <uax:String>http://opcfoundation.org/UA/</uax:String>
      </uax:ListOfString>
    </Value>
  </UAVariable>
</UANodeSet>