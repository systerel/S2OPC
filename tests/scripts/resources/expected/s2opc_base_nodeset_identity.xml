../s2opc_base_nodeset_identity.xml